src/main/java/org/fundacion/jala/converter/view/compiler/ProjectTab.java

```diff
@@ -176,8 +196,8 @@ public void changeTitle(final int numberOfClicks) {
      *
```

Comment:
Maybe it should be better to say "Gets the plus button to access it" in this java doc

Reply:
Solved!